app/(tabs)/notice.tsx

```diff
@@ -1,31 +1,72 @@
-import { StyleSheet } from 'react-native';
-
-import EditScreenInfo from '../../components/EditScreenInfo';
+import React, { useState } from 'react';
+import { StyleSheet, TextInput, useColorScheme, SafeAreaView, ScrollView } from 'react-native';
 import { Text, View } from '../../components/Themed';
 
 export default function NoticeScreen() {
+  const [houseRules, setHouseRules] = useState('');
+  const users = [
+    { name: 'Test Bruger', phone: '12 34 56 78', email: 'testbruger42@gmail.com' },
+    { name: 'Andreas Alstrup', phone: '87 65 43 21', email: 'andreas.alstrup@gmail.com' },
+  ];
   return (
-    <View style={styles.container}>
-      <Text style={styles.title}>Notice Board</Text>
-      <View style={styles.separator} lightColor="#eee" darkColor="rgba(255,255,255,0.1)" />
-      <EditScreenInfo path="app/(tabs)/notice.tsx" />
-    </View>
+    <SafeAreaView style={styles.container}>
+      <ScrollView>
+        <View>
+          <Text style={styles.title}>Household Rules</Text>
+          <TextInput
+            style={styles.input}
+            placeholder="Click to make rules"
+            value={houseRules}
+            onChangeText={(text) => setHouseRules(text)}
+            editable
+            multiline
+          />
+          <Text style={styles.title}>Contact Information</Text>
+          {users.map((user, index) => (
+            <View key={index} style={styles.userCard}>
+              <Text style={styles.userName}>{user.name}</Text>
+              <Text>Phone: {user.phone}</Text>
+              <Text>Email: {user.email}</Text>
+            </View>
+          ))}
+        </View>
+      </ScrollView>
+    </SafeAreaView>
   );
 }
 
 const styles = StyleSheet.create({
   container: {
     flex: 1,
-    alignItems: 'center',
-    justifyContent: 'center',
+    backgroundColor: 'white',
```

Comment:
This makes color also white when dark mode is on, maybe either remove this line entirely or make black when dark mode is on if you want to keep the full white background color

Reply:
We ditch dark mode.